src/components/AddItems.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import firebase from '../lib/firebase';
+
+const db = firebase.firestore().collection('shopping_list');
+
+const arrayUnion = firebase.firestore.FieldValue.arrayUnion;
+
+const AddItemsToList = () => {
+  const [shoppingListItemName, setShoppingListItemName] = useState('');
+  const [daysLeftForNextPurchase, setDaysLeftForNextPurchase] = useState(7);
+
+  const shoppingListItemNameHandler = (event) => {
+    setShoppingListItemName(event.target.value);
+  };
+  const daysLeftForNextPurchaseHandler = (event) => {
+    setDaysLeftForNextPurchase(parseInt(event.target.value));
+  };
+  function submitShoppingListItemHandler(event) {
+    event.preventDefault();
+
+    localStorage.setItem('token', 288838881888283);
```

Comment:
Am I right that when we merge both PRs we can delete this line, as the real token should be set by our PR?

Reply:
Exactly, that line is just to be used to test our functionality. it will be taken out before merging issue 4.

Reply:
This code looks so well structured and readable, great work !

Reply:
Yes yes, I totally agree Julia. Variables are named so well. It reads very well and is self-explanatory.

Reply:
An additional note: I see that the other team is using 'newToken' as their token name so make sure to coordinate with them before merging :)